GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Submodules.cs

```diff
@@ -488,9 +493,11 @@ private string GetNodeRelativePath(GitModule topModule, SubmoduleNode node)
                 foreach (var node in submoduleNodes)
                 {
                     var parts = GetNodeRelativePath(topModule, node).Split(Delimiters.ForwardSlash);
+
```

Comment:
I don't see why an empty line should be added here. `parts` is used only in the `for` loop below. It belongs to the loop.
The same applies to `path` below. I have not checked all the other added empty lines.
Thoughts, @gitextensions/git-extensions-source?

Reply:
For me, it first come down to 'digestible' chunks that I can easily read, then next chunk.
What belongs together is next level.

Some occurrences below belong together with this reasoning, some should have the newline.
Not a strong preference for me.

Reply:
@mstv I noticed you applying a few formattings during your last review, which made me go through some of the files I've touched in this PR, apply the pattern I recognized in your suggestions and add some new lines where I personally prefer them - mainly in front of `if/else` branches and loops. If you or other maintainers have a different preference there though, I'm happy to revert the new lines.

Reply:
My _personal_ preference is avoid excessive blank lines in situations like "get a variable on first line, use the variable in a condition or a loop on the following line", as in my view these are logically grouped. However I'm not aware of any "best practice" related to situations like this.

Reply:
@mstv @RussKie I understand your preferences and also that you don't feel strong enough about them to ask me to revert the excess whitespace changes - so I'll leave them as they were for now.